### ISSUE TITLE
Does lity support centos 7?

### ISSUE THREAD

**Issue by xxlunch** [CyberMiles/lity#9]
Does lity support centos 7? I am prompted to not support the Linux distribution when I install it.
![f73eb734091d0d6ce52eb9c85e29abb8](https://user-images.githubusercontent.com/44078281/47331430-8c390200-d6ae-11e8-8b05-18a5f72877ee.jpg)

**Comment by hydai** [comment]
Yes, Lity supports CentOS. You can find the section related to CentOS [here](https://github.com/CyberMiles/lity/blob/lity/scripts/install_deps.sh#L353).

What is your version of CentOS?

**Comment by xxlunch** [comment]
centos 7

**Comment by hydai** [comment]
What is your minor version of centos7?
I mean the whole version tag like `centos7.5.1804`. I need these information to reproduce your environment and test.

**Comment by hydai** [comment]
Dear @xxlunch,

Unfortunately,  Lity (and its upstream solidity) only supports CentOS6. We will figure out how to install the dependencies on CentOS7.
